packages/anoncreds/src/AnonCredsModuleConfig.ts

```diff
@@ -9,6 +12,18 @@ export interface AnonCredsModuleConfigOptions {
    * A list of AnonCreds registries to make available to the AnonCreds module.
    */
   registries: [AnonCredsRegistry, ...AnonCredsRegistry[]]
+
+  /**
+   * Maximum credential number per revocation registry
+   * @default 1000
+   */
+  maximumCredentialNumberPerRevocationRegistry?: number
```

Comment:
I think I've added this because initially I wanted to do an 'automatic approach' like ACA-Py and then switched to a manual one. So for the current code this setting does not make any sense.